Makefile

```diff
@@ -48,12 +48,21 @@ go-deps-clean:
 	$(RM) -r vendor
 
 sonic-telemetry: $(GO_DEPS)
+ifeq ($(CROSS_BUILD_ENVIRON),y)
+	$(GO) build -o ${GOBIN}/telemetry -mod=vendor $(BLD_FLAGS) github.com/Azure/sonic-telemetry/telemetry
```

Comment:
what is the difference between "go build -o ${GOBIN}/telemetry" and "go $(GO) install"?

Reply:
For the build purpose these two commands are the same. They both build the package and the result is placed in the same output directory. Somehow when cross-compiling "go install" command fails. So I replaced it with analogous build command that works fine.